_docs/individual-payments.md

```diff
@@ -440,6 +465,67 @@ orange
 
 {% include corridors/xof-bank.md recipient_type='individual' %}
 
+# Cameroon
+
+## XAF::Mobile
+
+For mobile payouts to Cameroon, please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "mobile_provider": "orange", // lowercase, see provider values below
+  "phone_number": "674044436", // mobile number in local country format
+  "country": "CM" // "CM" for Cameroon
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xaf-mobile-details" raw=data-raw %}
+
+The valid `mobile_provider` values for Cameroon are:
+
+{% capture data-raw %}
+```
+orange
+mtn
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="xaf-mobile-providers" raw=data-raw %}
+
+# Guinea
+
+## GNF::Mobile
+
+For mobile payouts to Guinea, please use:
+
+{% capture data-raw %}
+```javascript
+"details": {
+  "first_name": "First",
+  "last_name": "Last",
+  "mobile_provider": "mtn", // lowercase, see provider values below
+  "phone_number": "444044436" // mobile number in local country format
+}
+```
+{% endcapture %}
+
+{% include language-tabbar.html prefix="gnf-mobile-details" raw=data-raw %}
+
+The valid `mobile_provider` values for Guinea are:
+
+{% capture data-raw %}
+```
+orange
```

Comment:
GNF Orange is currently unavailable through Intouch - it'll be coming later. Probably best to remove this for now, though, as we don't have an ETA (as far as I know).